app/queue/ingress.py

```diff
@@ -93,7 +94,9 @@ def add_host(host_data):
             logger.info("Attempting to add host...")
             input_host = deserialize_host(host_data)
             staleness_timestamps = Timestamps.from_config(inventory_config())
-            (output_host, add_results) = host_repository.add_host(input_host, staleness_timestamps)
+            (output_host, add_results) = host_repository.add_host(
+                input_host, staleness_timestamps, fields=DEFAULT_FIELDS + ("tags", "system_profile")
```

Comment:
I wouldn’t be afraid of naming the _fields_ value.
```python
EGRESS_HOST_FIELDS = DEFAULT_FIELDS + ("tags", "system_profile")
```
```suggestion
                input_host, staleness_timestamps, fields= EGRESS_HOST_FIELDS)
```

Reply:
:+1: